examples/benchmark_problems/Neutron/ENGINX236516_vanadium_bank1_10brk.txt

```diff
@@ -1,7 +1,7 @@
 # FitBenchmark Problem
-# An example of fitting of a spline background on a Vanadium run from ENGIN-X
+software = 'Mantid'
 name = 'ENGINX 236516 vanadium, bank 1, 10 brk'
+description = 'An example of fitting of a spline background on a Vanadium run from ENGIN-X'
 input_file = 'ENGINX236516_vanadium_bank1.txt'
-function = 'name=BSpline, Order=3, NBreak=10, StartX=0.0, EndX=5.4'
-fit_parameters = {'StartX': 0, 'EndX': 5.4}
-description = ''
+function = 'name=BSpline, Order=3, NBreak=10'
```

Comment:
It seems that the [bspline](https://www.mantidproject.org/BSpline) function in mantid needs to have `StartX` and `EndX` as attributes. 
From setting the function to `function = 'name=BSpline, Order=3, NBreak=10, StartX=0.0, EndX=5.4'` in the branch gives (which are the same as master):
![with](https://user-images.githubusercontent.com/15614849/72508728-2221f500-383e-11ea-8174-a82af5cc8279.png)
and without gives:
![without](https://user-images.githubusercontent.com/15614849/72508872-5c8b9200-383e-11ea-9a84-0e783fddca3a.png)